database/sql/V068__create_jit_orders.sql

```diff
@@ -0,0 +1,34 @@
+-- JIT orders are stored in a separate table which contains data related only to JIT orders.
```

Comment:
Maybe add why that is (since there are some fields on the regular orders table that don't make sense for jit, e.g. xyz.

Reply:
added